lib/disconnect.js

```diff
@@ -0,0 +1,18 @@
+module.exports = disconnect
+
+var Promise = require('lie')
+
+/**
+ * disconnects local and remote database
+ *
+ * @return {Promise}
+ */
+function disconnect (state) {
+  if (state.replication) {
+    state.replication.cancel()
```

Comment:
Great catch! This didn’t cause problems so far, so I don’t think it’s critical, but I’ll make a follow up issues for it so we don’t forget to fix it